recipes/zlib-ng/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+from io import StringIO
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "test_package")
+        buffer = StringIO()
+        try:
+           self.run(bin_path, run_environment=True, output=buffer)
+           self.output.success("enough.c test passed")
+        except: # print output of test in case of error
+            self.output.error(buffer.getvalue())
```

Comment:
this is our usual template

```suggestion
import os
from conans import ConanFile, CMake, tools


class TestPackageConan(ConanFile):
    settings = "os", "compiler", "build_type", "arch"
    generators = "cmake", "cmake_find_package_multi"

    def build(self):
        cmake = CMake(self)
        cmake.configure()
        cmake.build()

    def test(self):
        if not tools.cross_building(self.settings):
            bin_path = os.path.join("bin", "test_package")
            self.run(bin_path, run_environment=True)
```